arm-web/2016-06-01/connectionGateways.json

```diff
@@ -400,27 +470,96 @@
         "type": {
           "type": "string",
           "readOnly": true,
-          "description": "Gets the status."
+          "description": "Gets the type."
         }
       },
       "description": "The gateway installation properties."
     },
-    "GatewayListResult": {
+    "ConnectionGatewayInstallationReferenceProperties": {
+      "type": "object",
+      "properties": {
+        "location": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets the location information."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets the display name of the installation."
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The id of the installation."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets the type."
+        }
+      },
+      "description": "The gateway installation reference properties."
+    },
+    "ConnectionGatewayReferenceProperties": {
```

Comment:
this model contains the properties that are generally part of an ARM resource, but seems to be used as a properties set, is this the intent?